src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1209,7 +1209,13 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _RestorePointerCursorHandlers(*this, nullptr);
 
         const auto point = args.GetCurrentPoint(*this);
-
+        // GH#10329 - we don't need to handle horizontal scrolls. Only vertical ones.
+        // So filter out the horizontal ones.
+        if (point.Properties().IsHorizontalMouseWheel())
+        {
+            return;
+        }
+        
         auto result = _interactivity.MouseWheel(ControlKeyStates{ args.KeyModifiers() },

```

Comment:
actually, quick question: it looks like this'll suppress the `_core->SendMouseEvent` in ControlInteractivity, is that ok? We don't want the VT mouse mode data anymore?

Reply:
not for horizontal wheel we don't.